### DISCUSSION TITLE
Add example Mermaid invocation in Wiki documentation

### DISCUSSION THREAD

**Discussion by rnickle** [lxndrblz/anatole#416] [Ideas]
Thank you for the excellent theme.

Very simply, in the Mermaid section [https://github.com/lxndrblz/anatole/wiki/%F0%9F%93%84-Content-Settings#enabling-the-mermaid-diagram](enabling the mermaid diagraml), add a very simple example invocation such as:

```text
```mermaid
graph TD;
    A-->B;
    A-->C;
    B-->D;
    C-->D;
```

Also, it seems clear from reading the source and from experimentation that while the Wiki suggests that you must put this block in _config.toml_, it actually seems to belong in _params.toml_ (or the equivalent params block):

```text
[mermaid]
enable = true
```

**Comment by lxndrblz** [comment]
@rnickle Thanks for reaching out. I've updated the relevant section in the wiki:
https://github.com/lxndrblz/anatole/wiki/%F0%9F%93%84-Content-Settings
